server/base/src/main/java/org/apache/accumulo/server/security/SecurityUtil.java

```diff
@@ -115,13 +115,14 @@ public static String getServerPrincipal(String configuredPrincipal) {
   /**
    * Start a thread that periodically attempts to renew the current Kerberos user's ticket.
    *
-   * @param conf Accumulo configuration
    * @param ugi The current Kerberos user.
    * @param renewalPeriod The amount of time between attempting renewals.
    */
-  static void startTicketRenewalThread(AccumuloConfiguration conf, final UserGroupInformation ugi,
-      final long renewalPeriod) {
-    Threads.createThread("Kerberos Ticket Renewal", () -> {
+  static void startTicketRenewalThread(final UserGroupInformation ugi, final long renewalPeriod) {
+    // TODO KEVIN RATHBUN this renewal seems like a critical task of any process running it, as not
+    // renewing the ticket would probably lead to authentication problems. This thread is also only
+    // created once.
+    Threads.createCriticalThread("Kerberos Ticket Renewal", () -> {
```

Comment:
this renewal seems like a critical task of any process running it, as not renewing the ticket would probably lead to authentication problems. This thread is also only created once.